src/pages/Messages.tsx

```diff
@@ -33,6 +35,8 @@ const styles = {
 };
 
 const Messages = () => {
+  const [messageModal, setMessageModal] = useState(false);
```

Comment:
Nitpick, what do you think about changing this to "showMessageModal"? I think it's a bit more descriptive